.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:a1cf4c44-6035-470e-b080-fd4088b2dc79
```

Comment:
I don't understand why any of the changes under `.vagrant` are necessary in this PR.